### PR TITLE
Lab2- Group8

### PR DESCRIPTION
**Pull request by FLatacela**
Francisco Latacela
Goutam Kundu

### CODE REVIEW

**Comment by khatchad** [comment]
Hm, there’s two Java files?

**Comment by FLatacela** [comment]
Good morning
I just checked and I think i did a mistake when i uploaded to github from netbeans But ths last file named coins/src/coins is my last version

Sorry for this inconvenient

It will affect our grade?
Sent from Outlookhttp://taps.io/outlookmobile

On Thu, Mar 5, 2015 at 7:54 AM -0800, "Raffi Khatchadourian" <notifications@github.com<mailto:notifications@github.com>> wrote:

Hm, there's two Java files?

##

**Comment by khatchad** [comment]
No I just need to know which file to grade. Thanks!

**Comment by FLatacela** [comment]
file named coins/src/coins

Sent from Outlookhttp://taps.io/outlookmobile

On Thu, Mar 5, 2015 at 2:10 PM -0800, "Raffi Khatchadourian" <notifications@github.com<mailto:notifications@github.com>> wrote:

No I just need to know which file to grade. Thanks!

##

**Comment by khatchad** [comment]
Thanks.

File: CoinsChange/src/coinschange/CoinsChange.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (khatchad) [inline CoinsChange/src/coinschange/CoinsChange.java:1]: Ignoring this file.

File: coins/src/coins/Coins.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * @authors: Francisco Latacela and Goutam Kundu
+ * @version 03/04/2015
+ */
+public class Coins {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO User input and declaring variables for d user amount of change in coins
```
Comment (khatchad) [inline coins/src/coins/Coins.java:22]: Please remove "TODO" prior to submission.

**Comment by khatchad** [comment]
What happened to the project files?

**Comment by FLatacela** [comment]
I just uploaded the java files mwithout the project files by mistake because i did not know how to do it
But now i know where i was doing my mistake when i pull to the github i just was doing the java files

Sent from Outlookhttp://taps.io/outlookmobile

On Sat, Mar 7, 2015 at 7:34 PM -0800, "Raffi Khatchadourian" <notifications@github.com<mailto:notifications@github.com>> wrote:

What happened to the project files?

##

**Pull request closed without merging**